doc/api/vm.markdown

```diff
@@ -301,6 +301,33 @@ e.g. `(0,eval)('code')`. However, it also has the following additional options:
 - `timeout`: a number of milliseconds to execute `code` before terminating
   execution. If execution is terminated, an [`Error`][] will be thrown.
 
+## Example: Run a Server within a VM
+
+Since the context of `.runInThisContext()` actually means the V8 context, you
+will need to apply additional objects to it. Say, you want to run a simple
+web server based on the `http` module, you can achieve this by doing something
+like this:
+
+```js
+'use strict'
+const vm = require('vm')
+
+let code = `
+const http = require('http');
+
+http.createServer( (request, response) => {
+  response.writeHead(200, {'Content-Type': 'text/plain'});
+  response.end('Hello World\\n');
+}).listen(8124);
+
+console.log('Server running at http://127.0.0.1:8124/');
+`
+
+let wrap = require('module').wrap(code)
```

Comment:
Please make valuable comments that bring the discussion forth not back. Any other way then than this and eval?